nibabel/freesurfer/tests/test_io.py

```diff
@@ -55,21 +56,31 @@ def test_geometry():
     assert_equal(coords.shape[0], faces.max() + 1)
 
     # Test quad with sphere
```

Comment:
Also, a little out of topic, but I haven't found `lh.sphere` files in the quad format. I don't think we're actually testing quad files at all.